app/src/sharedTest/java/org/oppia/app/splash/SplashActivityTest.kt

```diff
@@ -26,19 +72,149 @@ class SplashActivityTest {
     SplashActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
   )
 
-  @Before
-  fun setUp() {
-    Intents.init()
+  @Test
+  fun testSplashActivity_initialOpen_routesToOnboardingActivity() {
+    activityTestRule.launchActivity(null)
+    intended(hasComponent(OnboardingActivity::class.java.name))
   }
 
   @Test
-  fun testSplashActivity_initialOpen_routesToHomeActivity() {
-    activityTestRule.launchActivity(null)
-    intended(hasComponent(ProfileActivity::class.java.name))
+  fun testSplashActivity_secondOpen_routesToChooseProfileActivity() {
+    simulateAppAlreadyOpened()
+    launch(SplashActivity::class.java).use {
+      intended(hasComponent(ProfileActivity::class.java.name))
+    }
   }
 
-  @After
-  fun tearDown() {
-    Intents.release()
+  @Test
+  fun testSplashActivity_secondOpen_profileChooserScreenContentVerified() {
```

Comment:
Remove this test case, the content is not related to splash activity, this is a part of ProfileActivityTest

Reply:
Removed